addons/linecompass/functions/fnc_cacheLoop.sqf

```diff
@@ -33,6 +33,8 @@ if !(isNil "diwako_dui_special_track" && { diwako_dui_special_track isEqualType
     _unitsToRender append diwako_dui_special_track;
 };
 
+_unitsToRender = _unitsToRender - [objNull];
```

Comment:
might as well include that into line 31, one less assignment of the variable needed then.

Reply:
We can't because we appened diwako_dui_special_track afterward.